content/getting-started/sections/why-atom.md

```diff
@@ -3,7 +3,26 @@ title: Why Atom?
 ---
 ### Why Atom?
 
+####  About ATOM  
+   
+ATOM is essentially GitHub’s hackable text editor. It is a free open-source text editor and source code editor for macOS, Linux, and Microsoft Windows with support for plug-ins written in Node.js, and embedded Git Control, developed by GitHub. ATOM is based on Electron, an open source library developed by GitHub for building cross-platform desktop applications with HTML, CSS, and JavaScript.
+
+
+ ![Image of ATOM editor](https://github.com/vidyutgharpure/flight-manual.atom.io/blob/master/Atom-editor.png)  
+
+
+#### Comparison of ATOM with other text editors 
+
 There are a lot of text editors out there; why should you spend your time learning about and using Atom?
+Let us compare ATOM with a few other popular text editors. 
+
+&nbsp;   | Sublime       | TextMate      | Emacs         | Vim           | ATOM     
```

Comment:
General comments on the table:
- Capitalize all sentence fragments in tables ("Steep learning curve")
- From a usability/tech writing standpoint, I'm not sure what value this table adds. The metrics this table compares ("convenience", "approachability") are inherently hard to define in a quantitative matter and therefore difficult to draw meaningful comparison to just using a table. But I could be wrong!